pkgs/by-name/lm/lm4flash/package.nix

```
@@ -0,0 +1,56 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  libusb1,
+  pkg-config,
+  lib,
+  writeText,
+}:
+let
+  stellaris-udev-rules = writeText "61.stellpad.rules" ''
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="1cbe", ATTRS{idProduct}=="00fd", MODE="0666"
+  '';
+in
+stdenv.mkDerivation rec {
+  pname = "lm4flash";
+  version = "0.1.3";
+  src = fetchFromGitHub {
+    owner = "utzig";
+    repo = "lm4tools";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
nit: this alternative was recently added to avoid accidentally fetching a branch with the same name as the tag